src/ucai/core/databricks.py

```diff
@@ -211,6 +212,34 @@ def create_function(
             return self.client.functions.create(function_info)
         raise ValueError("Either function_info or sql_function_body should be provided.")
 
+    @override
+    def create_python_function(self, *, func: Callable, func_comment: str, catalog: str, schema: str) -> "FunctionInfo":
```

Comment:
```suggestion
    def create_python_function(self, *, func: Callable[..., Any], func_comment: str, catalog: str, schema: str) -> "FunctionInfo":
```

Reply:
good idea! Will update to make it more clear